### PR TITLE
Add Valentine's Day 2024 event

### LINKED ISSUE #550: Add Valentines Day 2024 event
### Describe your idea / enhancement

Get animation from Discord, stats from Twitter post.

### How would this idea / enhancement be beneficial?

Document the event.

### PR DESCRIPTION
**Pull request by MattIPv4**
## Describe your changes

Resolves #550

## Notes for testing your change

Description makes sense, video is correct, stats are correct.

### CODE REVIEW

File: apps/website/src/pages/events/index.tsx [hunk 1]
```diff
@@ -33,6 +35,53 @@ type Event = {
...
+          hand-crafted by Maya, were distributed to random raffle winners from
+          those who donated $25 or more.
+        </p>
+        <p>
+          We also revealed a new ambassador during the event, introducing
+          everyone to <Link href="/ambassadors/push-pop">Push Pop</Link>, our
+          Sulcata Tortoise. Live viewers also got to meet many of our other
+          ambassadors as we headed around the property to distribute
+          Valentine&apos;s-themed enrichment (crafted items to help encourage
+          natural behaviours like foraging) to them. Thanks to everyone that
```
Comment (pjeweb) [inline apps/website/src/pages/events/index.tsx:79]:
```suggestion
          natural behaviors like foraging) to them. Thanks to everyone who
```

**Review by pjeweb — APPROVED** [review]
looks good. the language tool had some suggestions, feel free to ignore them tho :)

File: apps/website/src/pages/events/index.tsx [hunk 2]
```diff
@@ -33,6 +35,53 @@ type Event = {
...
+      <>
+        <p>
+          Celebrating Valentine&apos;s Day once again, live viewers joined us
+          for a short fundraising stream. They were able to donate $25 or more
+          to get a signed postcard with artwork featuring the ambassadors to
+          commemorate the event, and an entry into the raffle. During the event,
+          we were able to raise $32,697 for the sanctuary, with 467 postcards
+          sent out to donors. The top 3 donors during the event were able to
+          pick the plushie they would like, and the remaining 15 plushies, all
+          hand-crafted by Maya, were distributed to random raffle winners from
```
Comment (pjeweb) [inline apps/website/src/pages/events/index.tsx:70]:
```suggestion
          handcrafted by Maya, were distributed to random raffle winners from
```

File: apps/website/src/pages/events/index.tsx (hunk 1 above)
Comment (MattIPv4) [reply apps/website/src/pages/events/index.tsx:79]: Ah yeah, US spelling is probably better

**Review by pjeweb — APPROVED** [review]

**Pull request merged**